src/components/Theme.js

```diff
@@ -1,7 +1,12 @@
 import getMuiTheme from 'material-ui/styles/getMuiTheme';
 import { white } from 'material-ui/styles/colors';
+import { deepConcat } from '../utils/objects';
 
-const muiTheme = getMuiTheme({
+export function customize(config) {
+  localStorage.setItem('customTheme', JSON.stringify(config));
```

Comment:
Probably a good idea do version the name, adding a prefix like `v1`.

That way we can invalidate previously set themes, just in case we ever need to change it.